core/common/src/main/java/alluxio/security/authorization/AccessControlList.java

```diff
@@ -600,4 +616,71 @@ public String toString() {
     List<String> entries = toStringEntries();
     return String.join(",", entries);
   }
+
+  /**
+   * This is a custom json serializer for AccessControlList class.
+   */
+  public static class AccessControlListSerializer extends StdSerializer<AccessControlList> {
+    private static final long serialVersionUID = -8523910728069876504L;
+
+    /**
+     * Creates a AccessControlListSerializer.
+     */
+    public AccessControlListSerializer() {
+      super(AccessControlList.class);
+    }
+
+    /**
+     * Serialize an AccessControlList object.
+     * @param accessControlList the ACL object to be serialized
+     * @param jsonGenerator json generator
+     * @param serializerProvider default serializer
+     * @throws IOException
+     */
+    @Override
+    public void serialize(AccessControlList accessControlList, JsonGenerator jsonGenerator,
+        SerializerProvider serializerProvider) throws IOException {
+      jsonGenerator.writeStartObject();
+      jsonGenerator.writeStringField("owner", accessControlList.getOwningUser());
```

Comment:
NIT: make `owner`, `owningGroup`, and `stringEntries` as constants.

Reply:
done